odin/assetlist.json

```diff
@@ -22,6 +22,27 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/odin/images/odin.svg",
 	"png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/odin/images/odin.png"
       }
-    }
+    },
+    {
+        "description": "GEO token for ODIN Protocol",
+        "denom_units": [
+          {
+            "denom": "mGEO",
+            "exponent": 0
+          },
+          {
+            "denom": "GEO",
+            "exponent": 6
+          }
+        ],
+        "base": "mGEO",
+        "name": "GEO",
+        "display": "geo",
```

Comment:
this display denom must match one of the listed denom units. Typically (but not always) the denom units would be all lowercase. if denom unit 'GEO' isn't changed to 'geo', then the display denom should be changed to 'GEO'.